docs/content-crag/concepts/assets/asset-materializations.mdx

```diff
@@ -97,9 +97,9 @@ There are a variety of types of metadata that can be associated with a materiali
 
 #### Example: Solid body
 
-```python file=concepts/assets/materialization_solids.py startafter=start_materialization_solids_marker_2 endbefore=end_materialization_solids_marker_2
-@solid
-def my_metadata_materialization_solid(context):
+```python file=concepts/assets/materialization_ops.py startafter=start_materialization_ops_marker_2 endbefore=end_materialization_ops_marker_2
```

Comment:
Example: Op Body

Reply:
This file is the assets materialization page that will be migrated separately, my edit is just changing the python filepath for the example